vue/components/ui/organisms/entity-show/entity-show.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+    <details-ripe
+        v-bind:title="_title"
+        v-bind:name="name"
+        v-bind:values="fields"
+        v-bind:item="entity"
+        v-bind:get-items="getEntities"
+        v-bind:options-items="optionsItems"
+        v-on:click:stats="onStatsButtonClick"
+        v-on:click:refresh="onRefreshButtonClick"
+        v-on:click:edit="onEditButtonClick"
+        v-on:click:delete="onDeleteButtonClick"
+    >
+        <slot v-bind:name="slot" v-for="slot in Object.keys($slots)" v-bind:slot="slot" />
+        <template v-for="slot in Object.keys($scopedSlots)" v-bind:slot="slot" slot-scope="scope">
+            <slot v-bind:name="slot" v-bind="scope" v-bind:entity="entity" />
+        </template>
+    </details-ripe>
+</template>
+
+<style lang="scss" scoped></style>
+
+<script>
+import { partMixin } from "../../../../mixins";
+
+export const EntityShow = {
+    name: "entity-show",
+    mixins: [partMixin],
+    props: {
+        name: {
+            type: String,
+            required: true
+        },
+        fields: {
+            type: Array,
+            required: true
+        },
+        getEntity: {
+            type: Function,
+            required: true
+        },
+        getEntityName: {
+            type: Function,
+            required: true
+        },
+        getEntities: {
+            type: Function,
+            required: true
+        },
+        title: {

```

Comment:
Title + breadcrumbs.
Title: pure override
Breadcrumbs: appends entity at the end

Reply:
Title > breadcrumbs